ML-Frameworks/pytorch-aarch64/requirements.txt

```diff
@@ -27,15 +28,17 @@ PyYAML~=6.0.2
 regex==2024.9.11
 requests~=2.32.3
 safetensors~=0.4.5
+sentencepiece==0.2.0  # Torchchat
```

Comment:
Ideally these would be `~=` to allow for patch releases to fix security issues in the future. Equally for setuptools, setting to `setuptools~=78.1.1` is equivalent to `setuptools>=78.1.1 <78.2`. It is kinda surprising that setuptools is here at all, but that might be a big job tracking that down.